docs/one-click-integrations.md

```diff
@@ -10,13 +10,41 @@ To set up the PagerDuty integration:
 
 1.  On Pulse, click **Integrations** and select **PagerDuty**.
 
-    ![PagerDuty integratio](images/pagerduty.png)
+    ![PagerDuty integration](images/pagerduty.png)
 
 1.  Fill in the necessary details:
 
-    -    **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
-    -    **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
+    - **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
+    - **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
 
 1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:
 
     ![PagerDuty webhook](images/pagerduty-webhook.png)
+
+## GitHub
+
+Pulse integrates directly with GitHub to receive data about deployments and changes.
+
+To set up the GitHub integration:
+
+1.  On Pulse, click **Integrations** and select **GitHub**.
+
+    ![GitHub integration](images/ghi-setup.png)
+
+1.  Click **Install GitHub App**
+
+1.  Follow the instructions on the GitHub UI to install the app in the desired organization.
```

Comment:
Here we could condense the two actions in a single step since the process is quite straightforward:

```suggestion
1.  Click **Install GitHub App** and follow the instructions on the GitHub UI to install the app in the desired organization.
```